src/components/CategoryCard/Category.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+function Category(Category) {
```

Comment:
props are small case 
```suggestion
function Category(category) {
```